src/net/l0/net_rx.c

```diff
@@ -33,7 +33,7 @@ int net_rx(struct sk_buff *skb) {
 	assert(skb != NULL);
 	assert(skb->dev != NULL);
 	if (skb->len < skb->dev->hdr_len) {
-		DBG(printk("net_rx: %p invalid length %zu\n", skb,
+		DBG(log_error("net_rx: %p invalid length %zu\n", skb,
```

Comment:
Of cause, I'm agree. DBG macro should switch to log_error. And in Mybuild file you should add option "log_level"